spec/components/summary_card/component_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe SummaryCard::View do
+  rows = {
+    'Character': "Lando Calrissian",
+    'Location': "In a galaxy",
+  }
+
+  before(:all) do
+    @result = render_inline(SummaryCard::View.new(title: "Lando Calrissian", heading_level: 6, rows: rows))
```

Comment:
```suggestion
    @result ||= render_inline(SummaryCard::View.new(title: "Lando Calrissian", heading_level: 6, rows: rows))
```

we probably want to memoise this value so it's not re-called every time it's referenced in the test?